components/Slider/src/MDCSlider.h

```diff
@@ -49,21 +49,140 @@ IB_DESIGNABLE
 
  Default color is blue.
  */
-@property(nonatomic, strong, null_resettable) UIColor *color UI_APPEARANCE_SELECTOR;
+@property(nonatomic, strong, null_resettable) UIColor *color NS_UNAVAILABLE;
```

Comment:
I'm not familiar with using NS_UNAVAILABLE on a property like this. Whats the advantage over just moving this to the .m file?

Reply:
This is a pretty old PR, and I should follow the deprecations schedule instead. You're right that this normally is only used for superclass APIs, and in this case we should just delete it long-term.